src/stellar/stellarEngine.js

```diff
@@ -475,8 +452,8 @@ export class StellarEngine extends CurrencyEngine {
 
     const networkFee = transaction.fee.toString()
     nativeAmount = bns.add(networkFee, nativeAmount) // Add fee to total
-    nativeBalance = bns.sub(nativeBalance, '10000000') // Subtract the 1 min XLM
-    if (bns.gt(nativeAmount, nativeBalance)) {
+    const nativeBalance2 = bns.sub(nativeBalance, '10000000') // Subtract the 1 min XLM
```

Comment:
Maybe `realNativeBalance` would have been a better name, but I get what this is doing.